### DISCUSSION TITLE
Trouble when launching

### DISCUSSION THREAD

**Discussion by Buramie** [zkxs/vrcx-optimal-time#45] [Q&A]
Hey there,

Already can say I'm not too good with that kind of stuff so it might be on my end. I followed the steps provided. Wasn't sure how to make the .toml but I copied the one from the repo and added my own info in it. 

When launching the executable it just opens a cmd and closes instantly, not too sure what to do from there.

**Comment by zkxs** [comment]
Are you launching it from an already opened terminal? You'll want to open a command prompt and then start it from that versus just running the executable directly from your file explorer.

**Reply by Buramie** [reply]
I did try that before but I guess not in the right, I did some search and am now launching it using the start command. It seems to display a lot more lines but still closes quite quickly.

Also not too sure how to get the data in the output.txt from what was written.

I added screenshots to make sure I did the stuff right.
![Screenshot 2023-09-15 134927](https://github.com/zkxs/vrcx-optimal-time/assets/60244726/17b45017-c611-4d9c-b7b8-e7cd969a9a06)
![Screenshot 2023-09-15 135131](https://github.com/zkxs/vrcx-optimal-time/assets/60244726/c651fa9a-8980-4157-9de8-b710c75c0803)
![Screenshot 2023-09-15 135338](https://github.com/zkxs/vrcx-optimal-time/assets/60244726/320d8e52-9678-4f17-b37e-89bcd3ddc8d2)

**Comment by zkxs** [comment]
The `start` command starts a separate window to run a command, which isn't what you want. Try this:

```batch
cd C:\Users\myusername\Documents\vrxtime
vrcx-optimal-time-windows-x64.exe config.toml > output.txt
```

It ought to dump the output into a text file for you.

**Reply by Buramie** [reply]
I see! Thanks for your time. I managed to make it work but sadly not with Reuben's viewer. I made my own graph in Excel instead which has maybe over 1+ year of data and 600 friends as of right now.

I'm guessing it does some sort of average, maybe could be nice to see the peak number ever recorded for each day as I'm pretty sure at times I had over 110 friends online.

![Picture1](https://github.com/zkxs/vrcx-optimal-time/assets/60244726/7d8d6f2a-d5e2-4002-b491-566d4bc5d5cf)

**Reply by zkxs** [reply]
Reuben's viewer expects a .csv file extension, so you might need to rename the `output.txt` from the command I gave you. I think it also might require `no_data_returns_zero = true` in the config? I don't remember exactly.

> I'm guessing it does some sort of average

https://github.com/zkxs/vrcx-optimal-time#how-does-it-work explains in decently specific language what it's doing. If you want absolute numbers you can set `normalize = false` in your config (this will make the data less useful for seeing trends but it'll give you the peaks you want).
